src/peer_channels.rs

```diff
@@ -480,13 +486,12 @@ pub async fn send_webxdc_realtime_data(ctx: &Context, msg_id: MsgId, data: Vec<u
 
 /// Leave the gossip of the webxdc with given [MsgId].
 pub async fn leave_webxdc_realtime(ctx: &Context, msg_id: MsgId) -> Result<()> {
```

Comment:
Note that when this is called, the app must guarantee that `send_webxdc_realtime*()` functions can't be called in parallel, otherwise they will race with `leave_webxdc_realtime()`